examples/java/build.gradle

```diff
@@ -52,6 +52,10 @@ configurations.sparkRunnerPreCommit {
 }
 
 dependencies {
+  implementation group: 'org.apache.beam', name: 'beam-sdks-java-core', version: '2.28.0'
+  implementation group: 'org.apache.beam', name: 'beam-runners-direct-java', version: '2.28.0'
```

Comment:
you can remove these two, as the project already depends on them (see lines 62-66)